### PR TITLE
Admin theme

### PR DESCRIPTION
**Pull request by jayesh-parmar**

### CODE REVIEW

**Comment by dineshram7660** [comment]
please update the main to this branch

File: resources/views/admin/layouts/nav.blade.php [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+            </button>
+            <!-- Search input -->
+            <div class="flex justify-center flex-1 lg:mr-32">
+                <div class="relative w-full max-w-xl mr-6 focus-within:text-purple-500">
+                    <div class="absolute inset-y-0 flex items-center pl-2">
+                        <svg class="w-4 h-4" aria-hidden="true" fill="currentColor" viewBox="0 0 20 20">
+                            <path fill-rule="evenodd" d="M8 4a4 4 0 100 8 4 4 0 000-8zM2 8a6 6 0 1110.89 3.476l4.817 4.817a1 1 0 01-1.414 1.414l-4.816-4.816A6 6 0 012 8z" clip-rule="evenodd"></path>
+                        </svg>
+                    </div>
+                    <input class="w-full pl-8 pr-2 text-sm text-gray-700 placeholder-gray-600 bg-gray-100 border-0 rounded-md dark:placeholder-gray-500 dark:focus:shadow-outline-gray dark:focus:placeholder-gray-600 dark:bg-gray-700 dark:text-gray-200 focus:placeholder-gray-500 focus:bg-white focus:border-purple-300 focus:outline-none focus:shadow-outline-purple form-input" type="text" placeholder="Search for projects" aria-label="Search" />
```
Comment (dineshram7660) [inline resources/views/admin/layouts/nav.blade.php]:
```suggestion
                    <input 
                    	class="w-full pl-8 pr-2 text-sm text-gray-700 placeholder-gray-600 bg-gray-100 border-0 rounded-md dark:placeholder-gray-500 dark:focus:shadow-outline-gray dark:focus:placeholder-gray-600 dark:bg-gray-700 dark:text-gray-200 focus:placeholder-gray-500 focus:bg-white focus:border-purple-300 focus:outline-none focus:shadow-outline-purple form-input" 
                    	type="text" 
                    	placeholder="Search for projects" 
                    	aria-label="Search" 
                    />
```

**Comment by jayesh-parmar** [comment]
👍

File: public/public/assets/js/init-alpine.js [hunk 2]
```diff
@@ -0,0 +1,62 @@
+function data() {
+  function getThemeFromLocalStorage() {
+    // if user already changed the theme, use it
```
Comment (dineshram7660) [inline public/public/assets/js/init-alpine.js]: remove comment

File: public/public/assets/js/init-alpine.js [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    if (window.localStorage.getItem('dark')) {
+      return JSON.parse(window.localStorage.getItem('dark'))
+    }
+
+    // else return their preferences
```
Comment (dineshram7660) [inline public/public/assets/js/init-alpine.js]: remove comment

File: public/public/assets/js/init-alpine.js [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+      this.isProfileMenuOpen = !this.isProfileMenuOpen
+    },
+    closeProfileMenu() {
+      this.isProfileMenuOpen = false
+    },
+    isPagesMenuOpen: false,
+    togglePagesMenu() {
+      this.isPagesMenuOpen = !this.isPagesMenuOpen
+    },
+    // Modal
```
Comment (dineshram7660) [inline public/public/assets/js/init-alpine.js]: remove comment

File: resources/views/admin/layouts/header.php [hunk 5]
```diff
@@ -0,0 +1,19 @@
+<!-- Desktop sidebar -->
```
Comment (dineshram7660) [inline resources/views/admin/layouts/header.php]: remove comment

File: resources/views/admin/layouts/nav.blade.php [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+                            class="absolute right-0 w-56 p-2 mt-2 space-y-2 text-gray-600 bg-white border border-gray-100 rounded-md shadow-md dark:border-gray-700 dark:text-gray-300 dark:bg-gray-700" aria-label="submenu">
+                            <li class="flex">
+                                 <a 
+                                     class="inline-flex items-center w-full px-2 py-1 text-sm font-semibold transition-colors duration-150 rounded-md hover:bg-gray-100 hover:text-gray-800 dark:hover:bg-gray-800 dark:hover:text-gray-200" 
+                                     href="#">
+                                    <svg class="w-4 h-4 mr-3" aria-hidden="true" fill="none" stroke-linecap="round" stroke-linejoin="round" stroke-width="2" viewBox="0 0 24 24" stroke="currentColor">
+                                        <path d="M16 7a4 4 0 11-8 0 4 4 0 018 0zM12 14a7 7 0 00-7 7h14a7 7 0 00-7-7z"></path>
+                                    </svg>
+                                    <span>Profile</span>
+                                </a>
```
Comment (dineshram7660) [inline resources/views/admin/layouts/nav.blade.php]: remove now

**Review by dineshram7660 — CHANGES_REQUESTED** [review]

**Comment by jayesh-parmar** [comment]
👍

File: resources/views/admin/layouts/header.php (hunk 5 above)
Comment (dineshram7660) [reply resources/views/admin/layouts/header.php]: this is not done

**Review by dineshram7660 — CHANGES_REQUESTED** [review]

File: resources/views/admin/layouts/header.php [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+<aside class="z-20 hidden w-64 overflow-y-auto bg-white dark:bg-gray-800 md:block flex-shrink-0">
+    <div class="py-4 text-gray-500 dark:text-gray-400">
+        <a class="ml-6 text-lg font-bold text-gray-800 dark:text-gray-200" href="{{route('dashboard)}}">
+            Inventory Management
+        </a>
+        <ul class="mt-6">
+            <li class="relative px-6 py-3">
+                <span class="absolute inset-y-0 left-0 w-1 bg-purple-600 rounded-tr-lg rounded-br-lg" aria-hidden="true"></span>
+                <a class="inline-flex items-center w-full text-sm font-semibold text-gray-800 transition-colors duration-150 hover:text-gray-800 dark:hover:text-gray-200 dark:text-gray-100" href="index.html">
```
Comment (dineshram7660) [inline resources/views/admin/layouts/header.php]: change url dashboard

File: resources/views/admin/layouts/header.php [hunk 8]
```diff
@@ -0,0 +1,19 @@
+<!-- Desktop sidebar -->
+<aside class="z-20 hidden w-64 overflow-y-auto bg-white dark:bg-gray-800 md:block flex-shrink-0">
+    <div class="py-4 text-gray-500 dark:text-gray-400">
+        <a class="ml-6 text-lg font-bold text-gray-800 dark:text-gray-200" href="{{route('dashboard)}}">
+            Inventory Management
+        </a>
```
Comment (dineshram7660) [inline resources/views/admin/layouts/header.php]: this url not working please check

**Review by dineshram7660 — CHANGES_REQUESTED** [review]

**Pull request merged**